Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/lucene/luceneindex.jsp

```diff
@@ -1,151 +1,171 @@
+                    });
 
-			$('#waitMe_ex')
-					.click(
-							function() {
-								run_waitMe(current_effect);
-								$
-										.ajax({
-											url : "${pageContext.servletContext.contextPath}/auth/indexConcepts",
-											type : "POST",
-											success : function(result){
-												setIndexValues(result);
-											},
-											error : function(result) {
-												var output = "<font color=\"red\">"
-														+ result.message
-														+ "</font>";
-												$("#div1").html(output);
-												waitMeClose();
-											}
-										});
-							});
+            function checkIndexerStatus() {
+                $
+                    .ajax({
+                        url: "${pageContext.servletContext.contextPath}/auth/getIndexerStatus",
+                        type: "POST",
+                        success: function(result, status,
+                            jqXHR) {
+                            // Poll here
+                            if (jqXHR.status == 202) {
+                                setTimeout(checkIndexerStatus,
+                                    5000);
+                            } else {
+                                setIndexValues(result);
+                                waitMeClose();
+                            }
+                        }
+                    });
+            }
 
-			function waitMeClose() {
-				$('.containerBlock > form').waitMe('hide');
-			}
+            function waitMeClose() {
+                $('.containerBlock > form').waitMe('hide');
+            }
 
-			function run_waitMe(effect) {
-				$('.containerBlock > form').waitMe({
-					effect : effect,
-					text : 'Please wait...',
-					bg : 'rgba(255,255,255,0.7)',
-					color : '#000',
-					maxSize : '',
-					source : 'img.svg',
-					onClose : function() {
-					}
-				});
-			}
-			
-			function setIndexValues(result) {
-				var output = "<center><b>" + result.message + " </b> </center>";
-				$("#div1").html(output);
-				$('#indexCount')
-						.html(result.indexedWordsCount);
-				$('#lastRun-1')
-						.html(result.lastRun);
-				waitMeClose();
-			}
-			
-		});
-	</script>
-</body>
+            function run_waitMe(effect) {
+                $('.containerBlock > form').waitMe({
+                    effect: effect,
+                    text: 'Please wait...',
+                    bg: 'rgba(255,255,255,0.7)',
+                    color: '#000',
+                    maxSize: '',
+                    source: 'img.svg',
+                    onClose: function() {}
+                });
+            }
 
+            function setIndexValues(result) {
+                var output = "<center><b>" + result.message +
+                    " </b> </center>";
+                $("#div1").html(output);
+                $('#indexCount').html(result.indexedWordsCount);
+                $('#lastRun-1').html(result.lastRun);
+                waitMeClose();
+            }
+        });
+    </script>
+</body>
```

Comment:
@jdamerow Please let me know if this formatting is fine? And also is there a way to format the jsp files in eclipse. I am facing some issues in formatting jsps in eclipse.